tempest/api/workloadmgr/base.py

```diff
@@ -2883,7 +2883,7 @@ def validate_snapshot_mount(
             cmd = "sudo su - root -c 'df -h'"
             stdin, stdout, stderr = ssh.exec_command(cmd, timeout=120)
             LOG.debug("df -h output: %s", stdout.read())
-            cmd = "sudo su - root -c 'ls -la " + file_path_to_search + "/Test_*/vda*'"
+            cmd = "sudo su - root -c 'ls -la " + file_path_to_search + "/Test_*/vda*/opt'"
```

Comment:
Is this path `opt` same in all tests ? Instead of hardcoding it, accept it as an optional parameter with `opt` as default value.

Reply:
updated